sitemetrics/templatetags/sitemetrics.py

```diff
@@ -92,4 +93,7 @@ def __init__(self, keycodes):
         self.template = template.loader.get_template('sitemetrics/sitemetrics.tpl')
 
     def render(self, context):
-        return self.template.render(template.Context({'keycodes': self.keycodes}))
+        context = template.Context({'keycodes': self.keycodes})
+        if django.VERSION >= (1, 8):
```

Comment:
I think this is extreme savings. Function call vs if statement. Just do it like you want. :)